### DISCUSSION TITLE
Headpat Haptics!

### DISCUSSION THREAD

**Discussion by giggletech** [vrchat-community/osc#167] [Show and tell]
Heya,

I made giggletech.io so people can put haptic points anywhere on the avatar

The current goal is to make it standalone, which would mean no need for external dependencies and a super smooth integration. However, I’m hitting a few roadblocks with Quest’s Open Sound Control (OSC) support. Finding solid documentation on how OSC handles standalone Quest devices has been a challenge – if anyone has resources or insights on this id love to chat!

-Sideways

**Comment by fisk1234ost** [comment]
Take a look at issue #129
